cockroachcloud/cmek-faq.md

```diff
@@ -15,11 +15,11 @@ Yes, the {{ site.data.products.dedicated }} clusters are encrypted by default by
 
 ## What steps should I take before enabling CMEK for a cluster?
 
-Contact your Cockroach Labs account team to enroll your organization in limited access for CMEK. CMEK can only be enabled on a private cluster, so your organization must also be enrolled in limited access for private clusters.
+CMEK can be enabled only on a private cluster. Refer to [Private Clusters](private-clusters.html).
```

Comment:
That's a preview + caching issue. I had the same thing, but it should be fixed now. it's because the Private Clusters topic was initially in the wrong sidebar.

The feature name is "private clusters" and it's a Dedicated-only feature. I'm going to go through and make sure that we are consistent. I'll update the page title too.

Reply:
Actually you are right, I had added this to the 23.1 sidebar but not the 22.2 sidebar. It should now be in the correct sidebar.